src/Backend/Core/Js/ckeditor/LICENSE.md

```diff
@@ -2,7 +2,7 @@ Software License Agreement
 ==========================

```

Comment:
Wouldn't it be a better idea to install CKEditor with yarn? https://www.npmjs.com/package/ckeditor.
Maybe outside the scope of this PR, but would be a huge improvement.

Reply:
I also think this would be a very big improvement, specially in terms of maintainability.
I just not sure of how you then add and setup extra plugins. I would have to look it up.